src/views/Home/Home.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { fetchDogs } from '../../services/dogsfetch';
+import './Home.css';
+
+export default function DogsList() {
+  const [dogs, setDogs] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchDogs();
+
+      setDogs(data);
+    };
+    fetchData();
+  }, []);
```

Comment:
Consider adding in a try/catch block to handle any errors that may be caused from our 3rd party supabase backend.